src/config/dynamic-theme-fixes.config

```diff
@@ -18671,6 +18675,14 @@ html[data-darkreader-scheme="dark"] img[src*="Eagle_with_fasces.svg"] {
     background-color: ${#BBBBBB} !important;
     mix-blend-mode: screen !important;
 }
+body.mediawiki.skin-monobook:has(> #globalWrapper) {
+    background-color: #f9f9f9 !important;
+    background-image: linear-gradient(#0000, #0000), url(https://upload.wikimedia.org/wikipedia/commons/f/fb/Monobook_skin_background.svg) !important;
+    filter: invert();
```

Comment:
As Dark Reader can be used in dark or light scheme filtering like this will force dark in both.

For better handling light and dark scheme I would remove invert from here.\
For solution check my next comment.